hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/signature/AWSSignatureProcessor.java

```diff
@@ -41,19 +39,26 @@
  * header. For more details refer to AWS documentation https://docs.aws
  * .amazon.com/general/latest/gr/sigv4-create-canonical-request.html.
  **/
-@RequestScoped
+
 public class AWSSignatureProcessor implements SignatureProcessor {
 
   private static final Logger LOG =
       LoggerFactory.getLogger(AWSSignatureProcessor.class);
 
   @Context
   private ContainerRequestContext context;
```

Comment:
`context` is no longer used.

Reply:
Thanks. Removed now unused `@Context` annotation and corresponding `ContainerRequestContext` variable.